.settings/.jsdtscope

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you add `.settings/` to `.gitignore` and remove this folder?

Also, eww, Eclipse? :trollface: